project/MimaExcludes.scala

```diff
@@ -82,6 +82,10 @@ object MimaExcludes {
             // SPARK-5166 Spark SQL API stabilization
             ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.ml.Transformer.transform"),
             ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.ml.Estimator.fit")
+          ) ++ Seq(
+            // SPARK-5315 Spark Streaming Java API returns Scala DStream
+            ProblemFilters.exclude[MissingMethodProblem](
+              "org.apache.spark.streaming.api.java.JavaDStreamLike.reduceByWindow")
```

Comment:
Hm, it doesn't make sense that Mima would complain since no methods have been removed. Can you double-check by trying a clean build without the exclusion? or else I am worried we're overlooking some obscure reason that this change still change the API, but I can't figure out why that could be. It could be a false positive in Mima.